data-migrations/migrations/20250723175849-rename_and_populate_createdbyemail_to_createdbyid.js

```diff
@@ -0,0 +1,40 @@
+export const up = async (db, client) => {
```

Comment:
1. we do not need to rename the fields.
2. let's fetch all users at once to reduce the DB query calls.